packages/controls/css/widgets-base.css

```diff
@@ -395,23 +395,13 @@
   line-height: var(--jp-widgets-inline-height);
   margin-right: var(--jp-widgets-inline-margin);
   margin-left: var(--jp-widgets-inline-margin);
-
-  /* from the fa class in FontAwesome: https://github.com/FortAwesome/Font-Awesome/blob/49100c7c3a7b58d50baa71efef11af41a66b03d3/css/font-awesome.css#L14 */
-  display: inline-block;
-  font: normal normal normal 14px/1 FontAwesome;
-  font-size: inherit;
-  text-rendering: auto;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
 }
 
 .widget-valid.mod-valid i:before {
-  content: '\f00c';
```

Comment:
Should this selector be for the icon instead? Same below.

Reply:
I guess it will still work since it is a child though.

Reply:
Maybe just remove the `:before` ?

Reply:
It needs to be before. That is where font awesome is adding the content via classes:
<img width="486" alt="Screen Shot 2020-03-17 at 9 02 24 AM" src="https://user-images.githubusercontent.com/192614/76875674-1ea2ee00-682e-11ea-821f-b675ba74a731.png">

To be sure, here it is working with this PR:

<img width="372" alt="Screen Shot 2020-03-17 at 9 02 36 AM" src="https://user-images.githubusercontent.com/192614/76875714-295d8300-682e-11ea-8f3e-65d37270b2a9.png">

Reply:
Actually, the fontawesome examples don't use :before, they inherit from the surrounding elements. It's simpler without the :before, so I removed it. Thanks!